package.nls.json

```
@@ -0,0 +1,205 @@
+	"command.pr.saveComment.title": "Save",
+	"command.pr.deleteComment.title": "Delete Comment",
+	"command.pr.resolveReviewThread.title": "Resolve Conversation",
+	"command.pr.unresolveReviewThread.title": "Unresolve Conversation",
+	"command.pr.signinAndRefreshList.title": "Sign in and Refresh",
+	"command.pr.configureRemotes.title": "Configure Remotes...",
+	"command.pr.refreshActivePullRequest.title": "Refresh",
+	"command.pr.markFileAsViewed.title": "Mark File As Viewed",
+	"command.pr.unmarkFileAsViewed.title": "Mark File As Not Viewed",
+	"command.pr.openReview.title": "Go to Review",
+	"command.pr.collapseAllComments.title": "Collapse All Comments",
+	"command.comments.category": "Comments",
+	"command.pr.editQuery.title": "Edit Query",
+	"command.pr.openPullsWebsite.title": "Open on GitHub",
+	"command.pr.resetViewedFiles.title": "Reset Viewed Files",
+	"command.issues.category": "GitHub Issues",
+	"command.issue.createIssueFromSelection.title": "Create Issue From Selection",
+	"command.issue.createIssueFromClipboard.title": "Create Issue From Clipboard",
+	"command.issue.copyGithubPermalink.title": "Copy GitHub Permalink",
+	"command.issue.copyGithubHeadLink.title": "Copy GitHub Head Link",
+	"command.issue.copyMarkdownGithubPermalink.title": "Copy GitHub Permalink as Markdown",
+	"command.issue.openGithubPermalink.title": "Open Permalink on GitHub",
+	"command.issue.openIssue.title": "Open Issue on GitHub",
+	"command.issue.copyIssueNumber.title": "Copy Number",
+	"command.issue.copyIssueUrl.title": "Copy Url",
+	"command.issue.refresh.title": "Refresh",
+	"command.issue.suggestRefresh.title": "Refresh Suggestions",
+	"command.issue.startWorking.title": "Start Working on Issue",
+	"command.issue.startWorkingBranchDescriptiveTitle.title": "Start Working on Issue and Checkout Topic Branch",
+	"command.issue.continueWorking.title": "Continue Working on Issue",
+	"command.issue.startWorkingBranchPrompt.title": "Start Working and Set Branch...",
+	"command.issue.stopWorking.title": "Stop Working on Issue",
+	"command.issue.stopWorkingBranchDescriptiveTitle.title": "Stop Working on Issue and Leave Topic Branch",
+	"command.issue.statusBar.title": "Current Issue Options",
+	"command.issue.getCurrent.title": "Get current issue",
+	"command.issue.editQuery.title": "Edit Query",
+	"command.issue.createIssue.title": "Create an Issue",
+	"command.issue.createIssueFromFile.title": "Create Issue",
+	"command.issue.issueCompletion.title": "Issue Completion Chosen",
+	"command.issue.userCompletion.title": "User Completion Chosen",
+	"command.issue.signinAndRefreshList.title": "Sign in and Refresh",
+	"command.issue.goToLinkedCode.title": "Go to Linked Code",
+	"command.issues.openIssuesWebsite.title": "Open on GitHub",
+	"welcome.github.login.contents": {
+		"message": "You have not yet signed in with GitHub\n[Sign in](command:pr.signin)",
+		"comment" : [
+			"Do not translate what's inside of (...). It is link syntax."

```

```suggestion
			"Do not translate what's inside of (...). It is link syntax.",
			"{Locked='](command:pr.signin)'}"
```
in the past translators have messed with the markdown syntax by adding a space between `]` and `(` so I've historically locked those.